CONTRIBUTING.md

```diff
@@ -67,8 +67,7 @@ that lacks a sign-off to this agreement will not be accepted by the OpenBao proj
 | James Butcher     | james@iotechsys.com                 |                                            | Brad Corrion (brad@iotechsys.com)       | IOTech Systems            | Member             |
 | Alain Nochimowski | alain.Nochimowski@viaccess-orca.com |                                            | Dan Ghita (dan.ghita@viaccess-orca.com) | Viaccess-Orca             | Member             |
 | Julian Cassignol  | jcassignol@wallix.com               |                                            | Cristian Popi (cpopi@wallix.com)        | Wallix                    | Member             |
-| Alexander Scheel  | ascheel@gitlab.com                  | [@cipherboy](https://github.com/cipherboy) |                                         | GitLab                    | Chair (non-voting) |
-
+| Alexander Scheel  | ascheel@gitlab.com                  | [@cipherboy](https://github.com/cipherboy) | Jocelyn Eillis (jeillis@gitlab.com)     | GitLab                    | Chair              |
```

Comment:
Hmm, I think you'll need to fix your rebase here. We have a [maintainer commit signature requirement](https://openbao.org/docs/rfcs/signed-commits/) and [workflow](https://github.com/openbao/openbao/blob/main/.github/workflows/verify-commits.yml) that means others can't merge unsigned commits from maintainers.